### ISSUE TITLE
pagingViewController.menuBackgroundColor didn't work at "v2.2.0"

### ISSUE THREAD

**Issue by Edward-Ryu** [rechsteiner/Parchment#459]

**Comment by simonzwicker** [comment]
Same problem here

**Comment by rechsteiner** [comment]
Hmm, this seems to work fine in the Example target using this:

```Swift
let pagingViewController = PagingViewController(viewControllers: viewControllers)
pagingViewController.menuBackgroundColor = .red
```

Could you share a bit more about your implementation?

**Comment by qcasxy** [comment]
> Hmm, this seems to work fine in the Example target using this:
> 
> ```swift
> let pagingViewController = PagingViewController(viewControllers: viewControllers)
> pagingViewController.menuBackgroundColor = .red
> ```
> 
> Could you share a bit more about your implementation?

The same problem occurred when I set the `menuItemSize`.
```
let pagingViewController = PagingViewController(viewControllers: viewControllers)
pagingViewController.menuItemSize = .fixed(width: 120, height: 60)
pagingViewController.menuBackgroundColor = .green
```

**Comment by rechsteiner** [comment]
Thanks @qcasxy! This should be fixed now: #465. Will release a new version soon, but for now you can target the `master` branch 🙂

**Comment by rechsteiner** [comment]
Version `v2.3.0` is available now with the fix 🙂
